zppy/templates/ocean_month.py

```diff
@@ -29,10 +29,10 @@
 tunits = "days since 0001-01-01 00:00:00"
 
 # Loop over year sets
-for y in range(start_yr, end_yr, 10):
+for y in range(start_yr, end_yr, ts_num_years):
 
     year1 = y
-    year2 = y + 10 - 1
+    year2 = y + ts_num_years - 1
```

Comment:
In #219 I set `year2 = min(y + 10 - 1, end_yr)` here as was suggested on #208. I'm assuming this change here would actually be able to close out #208 then?

Reply:
Note: the number of years still has to be divisible by the increment.